app/api/migrations/migrations/36-populate-mimetype-on-attachments/specs/fixtures.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
I think, given the size of the dataset, that it doesn't merit to have a fixtures file. Moving this data to the test file will make it much more readable.

Reply:
On it

Reply:
Done